src/mds/MDCache.cc

```diff
@@ -1414,6 +1423,31 @@ void MDCache::adjust_subtree_after_rename(CInode *diri, CDir *olddir, bool pop)
   show_subtrees();
 }
 
+// ===================================
+// Cache Autotuning
+
+
+/*
+ * If mds_cache_autotune is set to true, then adjust cache_memory_limit using the rss usage periodically as per the mds_memory_cache_resize_interval
+ */
+void MDCache::adjust_cache_memory_limit()
+{
+  static MemoryModel mm(g_ceph_context);
+  static MemoryModel::snap last;
+  utime_t next_tuning = ceph_clock_now();
```

Comment:
We try not to use `utime_t` anymore in favor of monotonic clocks. See usage in other code of `ceph::coarse_mono_clock`, e.g. in `src/common/DecayCounter.h`.

Reply:
Ack